src/chapters/setup-intro-data-types/exercises.rst

```diff
@@ -1,2 +1,152 @@
 Exercises: Data Types
 ======================
+
+Initial Setup
+-------------
+
+The steps here will walk you through setting up a repository that you’ll
+use to study example code, work on studios, and complete your first
+assignment of this unit.
+
+#. Create a *Fork* of `LaunchCodeEducation/java-web-dev-exercises <https://github.com/LaunchCodeEducation/java-web-dev-exercises>`__.
+   Do not clone and create a local version of this repo just yet. 
+   
+#. Open IntelliJ.
+
+   .. note::
+
+      If the app opens up to an existing project, select *IntelliJ >
+      Preferences > Appearance & Behavior > System Settings* and uncheck
+      *Reopen last project on startup*. (For Windows users: *File >
+      Settings > Appearance & Behavior > System Settings*.) Close and
+      Reopen IntelliJ.
+
+#. From the “Welcome to IntelliJ” dialog, select *Check out from Version
+   Control > Git*
+   **TODO: Get screenshots of this process.**
+#. Click the button on the lower left corner of the dialog to configure your Github account.
+#. The first time you use GitHub within IntelliJ, you’ll have to
+   configure IntelliJ to use your GitHub account. We recommend taking
+   the extra step of authenticating using a token. This takes only one
+   brief extra step, and will prevent you from having to update IntelliJ
+   settings if you ever change your GitHub password.
+#. From the URL dropdown options, select your fork of ``java-web-dev-exercises``, 
+   along with an appropriate source destination directory (i.e. a folder where you’ve stored other projects for this class).
+#. When asked “Would you like to create an IDEA project…” select *Yes*, 
+   and then accept all of the default options that are presented.
+
+With that, you’re ready to go!
+
+Here’s a video walking through the last 4 steps above:
+
+.. raw:: html
+
+   <div class="youtube-wrapper">
+
+.. raw:: html
+
+   <iframe width="560" height="315" src="https://www.youtube.com/embed/OPCaYVXRm_c" frameborder="0" allowfullscreen>
+
+.. raw:: html
+
+   </iframe>
+
+.. raw:: html
+
+   </div>
+
+Troubleshooting
+---------------
+
+ClassNotFoundException
+^^^^^^^^^^^^^^^^^^^^^^
+
+If you experience ``java.lang.ClassNotFoundException`` when trying to
+run code after setting up the project, follow these steps: 
+
+#. Select *File > Close Project*.
+#. From the IntelliJ modal window, click the *X* next to ``java-web-dev-exercises`` in the left-hand pane.
```

Comment:
Do the students know what "modal window" means? Is there an IntelliJ workspace overview presented in the videos, or will we need one in the chapter?

Even if the info is in the "Setup" page, a screenshot of the window (with an arrow for more direction) would be helpful.

Reply:
good call. Im adding screenshots here and changing the language